packages/dev/core/src/Maths/math.color.ts

```diff
@@ -135,58 +138,260 @@ export class Color3 {
+     * @returns the new Color3
+     */
+    public divide(otherColor: DeepImmutable<Color3>): this {
+        return new (this.constructor as Constructor<typeof Color3, this>)(this.r / otherColor.r, this.g / otherColor.g, this.b / otherColor.b);
+    }
+
+    /**
+     * Divides the current Color3 coordinates by the given ones and stores the result in the given color "result"
+     * @param otherColor defines the second operand
+     * @param result defines the Color3 object where to store the result
+     * @returns the result
+     */
+    public divideToRef<T extends Color3>(otherColor: DeepImmutable<Color3>, result: T): T {
+        return result.copyFromFloats(this.r / otherColor.r, this.g / otherColor.g, this.b / otherColor.b);
+    }
+
+    /**
+     * Divides the current Color3 coordinates by the given ones.
+     * @param otherColor defines the second operand
+     * @returns the current updated Color3
+     */
+    public divideInPlace(otherColor: DeepImmutable<Color3>): this {
+        this.r = this.r / otherColor.r;
+        this.g = this.g / otherColor.g;
+        this.b = this.b / otherColor.b;
         return this;
     }
 
+    /**
+     * Updates the current Color3 with the minimal coordinate values between its and the given color ones
+     * @param other defines the second operand
+     * @returns the current updated Color3
+     */
+    public minimizeInPlace(other: DeepImmutable<Color3>): this {
+        return this.minimizeInPlaceFromFloats(other.r, other.g, other.b);
+    }
+
+    /**
+     * Updates the current Color3 with the maximal coordinate values between its and the given color ones.
+     * @param other defines the second operand
+     * @returns the current updated Color3
+     */
+    public maximizeInPlace(other: DeepImmutable<Color3>): this {
+        return this.maximizeInPlaceFromFloats(other.r, other.g, other.b);
+    }
+
+    /**
+     * Updates the current Color3 with the minimal coordinate values between its and the given coordinates
+     * @param r defines the r coordinate of the operand
+     * @param g defines the g coordinate of the operand
+     * @param b defines the b coordinate of the operand
+     * @returns the current updated Color3
+     */
+    public minimizeInPlaceFromFloats(r: number, g: number, b: number): this {
+        if (r < this.r) {
+            this.r = r;
+        }
+        if (g < this.g) {
+            this.g = g;
+        }
+        if (b < this.b) {
+            this.b = b;
+        }
+        return this;
+    }
+
+    /**
+     * Updates the current Color3 with the maximal coordinate values between its and the given coordinates.
+     * @param r defines the r coordinate of the operand
+     * @param g defines the g coordinate of the operand
+     * @param b defines the b coordinate of the operand
+     * @returns the current updated Color3
+     */
+    public maximizeInPlaceFromFloats(r: number, g: number, b: number): this {
+        if (r > this.r) {

```

Comment:
use max instead if not slower ?

Reply:
Fixed in [157fc18](https://github.com/BabylonJS/Babylon.js/pull/13699/commits/157fc18cd530c9949d920475cb52ee5670b09539)